### PR TITLE
New Template: alcatel_sros_show_router_rsvp_interface

### PR DESCRIPTION
**Pull request by h4ndzdatm0ld**
- New Template Pull Request

alcatel_sros_show_router_rsvp_interface,
alcatel_sros, 
show router rsvp interface

Additional SROS template for Nokia/Alcatel

### CODE REVIEW

File: templates/alcatel_sros_show_router_rsvp_interface.textfsm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Value Required ACTIVE_SESSIONS (\d+|-)
+Value Required TOTAL_BW (\d+|-)
+Value Required RESV_BW (\d+|-)
+Value Required ADMIN_STATE (Up|Dwn|Down)
+Value Required OPER_STATE (Up|Dwn|Down)
+
+Start
+  ^=+
+  ^RSVP\s+Interfaces
+  ^Interface\s+Total\s+Active\s+Total\s+BW\s+Resv\s+BW\s+Adm\s+Opr -> Interface
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_router_rsvp_interface.textfsm:12]:
```suggestion
  ^Interface\s+Total\s+Active\s+Total\s+BW\s+Resv\s+BW\s+Adm\s+Opr\s*$$
```

File: templates/alcatel_sros_show_router_rsvp_interface.textfsm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ^\s+Sessions\s+Sessions\s+\(Mbps\)\s+\(Mbps\)
+  ^-+
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_router_rsvp_interface.textfsm:14]:
```suggestion
  ^-+ -> Interface
```

File: templates/alcatel_sros_show_router_rsvp_interface.textfsm [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  ^\s*$$
+  ^. -> Error
+
+Interface
+  ^${INTERFACE}\s*${TOTAL_SESSIONS}\s*${ACTIVE_SESSIONS}\s*${TOTAL_BW}\s*${RESV_BW}\s*${ADMIN_STATE}\s*${OPER_STATE} -> Record
+  ^\s+\S+
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_router_rsvp_interface.textfsm:20]:
```suggestion
```

File: templates/alcatel_sros_show_router_rsvp_interface.textfsm [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  ^Interfaces
+  ^=+
+  ^-+
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_router_rsvp_interface.textfsm:23]:
```suggestion
  ^-+ -> Done
```

File: templates/alcatel_sros_show_router_rsvp_interface.textfsm [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  ^\s*$$
+  ^. -> Error
```
Comment (jmcgill298) [inline templates/alcatel_sros_show_router_rsvp_interface.textfsm:25]:
```suggestion
  ^. -> Error

Done
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks @h4ndzdatm0ld the `\s+\S+` is a little too generic, but I believe these updates will match the lines per the test file

**Comment by h4ndzdatm0ld** [comment]
Updated changes and pulled on my test env and ran it against a device - updates look good

**Pull request merged**